packages/server/src/migrations/20230223T215019669Z-refactor.ts

```diff
@@ -0,0 +1,66 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<any>): Promise<void> {
+  await db.schema
+    .createTable('operations_new')
+    .addColumn('did', 'text', (col) => col.notNull())
+    .addColumn('operation', 'jsonb', (col) => col.notNull())
+    .addColumn('cid', 'text', (col) => col.notNull())
+    .addColumn('nullified', 'boolean', (col) => col.notNull())
+    .addColumn('createdAt', 'timestamptz', (col) =>
+      col.defaultTo(sql`current_timestamp`),
+    )
```

Comment:
Even with the `DEFAULT`, this should be `NOT NULL` (unless Kysely is doing that automatically).

Reply:
ah you're right 👍